snapcraft/internal/build_providers/_base_provider.py

```diff
@@ -246,10 +246,15 @@ def launch_instance(self) -> None:
             # what is on the host
             self._setup_snapcraft()
 
-    def _verify_base_change(self) -> None:
+    def _ensure_base(self) -> None:
         info = self._load_info()
         provider_base = info["base"] if "base" in info else None
         if self._base_has_changed(self.project.info.base, provider_base):
+            logger.warning(
```

Comment:
can you use `self.echoer.warn` here instead?

Reply:
Changed call to `self.echoer.warning`.

Reply:
hmm, I am not seeing this.

Reply:
Yep, sorry for the confusion. Merged commits from master and a rebase leftover made me think it was fixed elsewhere. It's there now.